test/unit/dialects/better-sqlite3.js

```diff
@@ -33,10 +39,44 @@ describe('better-sqlite3 unit tests', () => {
         },
       });
 
-      const result = await knexInstance.select(knexInstance.raw('2 + 2 as answer')).first().catch(err => err);
+      const result = await knexInstance
+        .select(knexInstance.raw('2 + 2 as answer'))
+        .first()
+        .catch((err) => err);
       console.dir(result);
       expect(result).to.be.an('error');
       expect(result.code).to.equal('MODULE_NOT_FOUND');
     });
   });
+
+  describe('readonly', () => {
+    it('should initialize the DB with the passed-in `readonly` option', async () => {
+      const knexInstance = knex({
+        client: 'better-sqlite3',
+        useNullAsDefault: true,
+        connection: {
+          filename: '/Users/erindees/src/knex/test/test.sqlite3',
```

Comment:
filename doesn't exists so unit test fails

Reply:
Whoops, meant to change those paths to relative before committing. That's what I get for pushing code when I'm tired. 😅 Fixed.